pkg/jsonnet/eval.go

```diff
@@ -31,6 +31,7 @@ type Opts struct {
 	ExtCode     InjectedCode
 	TLACode     InjectedCode
 	ImportPaths []string
+	EvalPattern string
```

Comment:
I know it was like this before, but could these get comments?

Reply:
I don't know what the others do. Care to do it yourself? :-)